frontends/election-manager/src/index.tsx

```diff
@@ -11,42 +11,52 @@ import {
   LocalStorage,
 } from '@votingworks/utils';
 import { Logger, LogSource } from '@votingworks/logging';
+import {
+  ErrorBoundary,
+  Prose,
+  QUERY_CLIENT_DEFAULT_OPTIONS,
+  Text,
+} from '@votingworks/ui';
 import { App } from './app';
 import { ElectionManagerStoreAdminBackend } from './lib/backends';
 import { ServicesContext } from './contexts/services_context';
+import { ApiClientContext, createApiClient } from './api';
 
-const queryClient = new QueryClient({
-  defaultOptions: {
-    mutations: { networkMode: 'always' },
-    queries: {
-      networkMode: 'always',
-
-      // Avoid re-fetching data unnecessarily on page navigation and rely on
-      // explicit cache invalidation calls when we mutate data:
-      staleTime: Infinity,
-    },
-  },
-});
 const storage = window.kiosk
   ? new KioskStorage(window.kiosk)
   : new LocalStorage();
 const logger = new Logger(LogSource.VxAdminFrontend, window.kiosk);
 const backend = new ElectionManagerStoreAdminBackend({ storage, logger });
+const apiClient = createApiClient();
+const queryClient = new QueryClient({
+  defaultOptions: QUERY_CLIENT_DEFAULT_OPTIONS,
+});
 
 ReactDom.render(
   <React.StrictMode>
-    <QueryClientProvider client={queryClient}>
+    <ErrorBoundary
+      errorMessage={
+        <Prose textCenter>
+          <h1>Something went wrong</h1>
+          <Text>Please restart the machine.</Text>
+        </Prose>
+      }
+    >
       <ServicesContext.Provider value={{ backend, logger, storage }}>
-        <App />
+        <ApiClientContext.Provider value={apiClient}>
```

Comment:
I moved the `ApiClientContext.Provider` up a level to `index.tsx` because I realized that, when it was in `App`, it was getting rendered twice in tests:

https://github.com/votingworks/vxsuite/blob/a51a57ee4f8ec5818f54b2ea1778cd9a5cc2547c/frontends/election-manager/test/render_in_app_context.tsx#L94-L102

I couldn't just remove the `ApiClientContext.Provider` from the test util because the `renderRootElement` helper is also used to render elements other than `App` in tests.

I think we'd ideally move all the wrappings in `index.tsx` into `App` to better mirror VxMark and VxScan (and update tests and test helpers accordingly), but that was out of scope for this PR

Relatedly, with the `QueryClientProvider` also in this file, I had to render the `ErrorBoundary` here, making it hard to write an error boundary test (because of the `ReactDom.render` in this file). But I did test manually

Reply:
Cool, that sounds fine for now. If you wanted you could leave a TODO comment, but not necessary